api/v1beta1/kafkacluster_types.go

```diff
@@ -544,7 +555,14 @@ type ExternalListenerConfig struct {
 	// If accessMethod is Nodeport and externalStartingPort is set to 0 then the broker IDs are not added and the Nodeport port numbers will be chosen automatically by the K8s Service controller
 	ExternalStartingPort int32 `json:"externalStartingPort"`
 	// configuring AnyCastPort allows kafka cluster access without specifying the exact broker
+	// If not defined, 29092 will be used for external clients to reach the kafka cluster
 	AnyCastPort *int32 `json:"anyCastPort,omitempty"`
+	// +kubebuilder:validation:Minimum=1024
```

Comment:
**Question**: sorry about being this pedantic, but is 1024 safe to use already? My impression was 1025 would be the first but could be wrong, haven't looked into it.

Reply:
Well known ports are [0,1023] (can't really find a agreed reference by the networking industry due to my limited knowledge), but a reference from wikipedia: https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers

And I've tested with "istioingress" ingress controller, 1024 works fine as the container port within the container

Reply:
Sounds good, thanks.